docs/sources/clients/promtail/stages/drop.md

```diff
@@ -0,0 +1,179 @@
+---
+title: drop
+---
+# `drop` stage
+
+The `drop` stage is a filtering stage that lets you drop logs based on several options. 
+
+It's important to note that if you provide multiple options they will be treated like an AND clause,
+where each option has to be true to drop the log.
+
+If you wish to drop with an OR clause, specify multiple drop stages.
+
+There are examples below to help explain. 
+
+## Schema
+
+```yaml
+drop:
+  # Name from extracted data to parse. If empty, uses the log message.
+  [source: <string>]
+  
+  # RE2 regular expression, if source is provided the regex will attempt to match the source
+  # If no source is provided, the regex will attempt to match the log line
```

Comment:
When possible, write in present tense. "attempts" rather than "will attempt." Please apply throughout.